README.md

```diff
@@ -69,11 +69,14 @@ You should now be able to run Cylc commands, e.g.:
 
 Start the Hub (JupyterHub gets installed with the "cylc" package):
 
-- `jupyterhub
-    --JupyterHub.spawner_class="jupyterhub.spawner.LocalProcessSpawner"
-    --Spawner.args="['-s', '${CONDA_PREFIX}/work/cylc-ui']"
+```sh
+cd "${CONDA_PREFIX}/work"
+jupyterhub \
+    --JupyterHub.spawner_class="jupyterhub.spawner.LocalProcessSpawner" \
+    --JupyterHub.logo_file="./cylc-ui/img/logo.png" \
+    --Spawner.args="['-s', './cylc-ui']" \
```

Comment:
I think perhaps we could use another directory? The `CONDA_PREFIX` is the location of the conda environment directory. That directory is similar to `virtualenv` & `venv`'s directory, where normally users expect the content to change when they run commands with `pip`, `conda`, `etc.

You could destroy a Conda environment which would delete the JupyterHub sqlite database too, which may cause issues to some users?

But I agree moving it to another directory would be good. I started it a few times inside `cylc-conda` for convenience and then had to remove the JupyterHub files.

Reply:
Destroying everything is (in my normal way of thinking) a good thing for a dev/test environment. 😈 (Just in case of incompatibility of these files between future JupyterHub versions?)

I do see your point, however. Any suggestion? Something like:
* $PREFIX/srv/cylc/
* $PREFIX/srv/cylc-hub/
* $PREFIX/srv/jupyterhub/

where $PREFIX would be $HOME for a normal user.

Reply:
Updated to go to `${HOME}/srv/cylc/` for now.